Content.Server/Kitchen/EntitySystems/ReagentGrinderSystem.cs

```diff
@@ -123,10 +125,8 @@ public override void Update(float frameTime)
                         if (solution.Volume > containerSolution.AvailableVolume)
                             continue;
 
-                        var dev = new DestructionEventArgs();
-                        RaiseLocalEvent(item, dev);
-
-                        QueueDel(item);
+                        if(!_destructible.DestroyEntity(item))
+                            continue;
```

Comment:
```suggestion
                        _destructible.DestroyEntity(item);
```
got lost in the sauce